components/common/listingCart/shared/ListingCartPriceInput.vue

```diff
@@ -21,19 +21,22 @@
 
 <script setup lang="ts">
 import { NeoButton } from '@kodadot1/brick'
-import { useVModel } from '@vueuse/core'
```

Comment:
auto imported?

Reply:
Yes looks like from Nuxt v7.2.0 auto-import is enabled with most utils in the @vueuse/nuxt build module. 👍

Reply:
Noted!